internal/resources/slo/data_source_slo_test.go

```diff
@@ -0,0 +1,23 @@
+package slo_test
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/grafana/terraform-provider-grafana/internal/testutils"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceSlo(t *testing.T) {
+	testutils.CheckCloudInstanceTestsEnabled(t)
+
+	resource.ParallelTest(t, resource.TestCase{
+		ProviderFactories: testutils.ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config:      testutils.TestAccExample(t, "data-sources/grafana_slo/data-source.tf"),
+				ExpectError: regexp.MustCompile(`No SLOs Exist`),
+			},
```

Comment:
You could add a test here that uses both the SLO resource example and the datasource example. That way, you can test that your datasource returns what your resource pushed